### ISSUE TITLE
MultiComboBox: A11Y | JAWS Screen reader - Repetitive announcements should be avoided

### ISSUE THREAD

**Issue by t3s3r4ct** [SAP/ui5-webcomponents-react#7265]
### Describe the bug

Note: This may be more appropriate as a consulting issue.

Using the screen reader JAWS 2025, there are redundant announcements when using the MultiComboBox component and items.

`Multiple selection mode` is announced for each item in the combo box list. But `Contains Multi-Selectable Items list` announcement is already made, leading to repetitive announcements.

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/inputs-multicombobox--docs

### Reproduction steps

1. Open the speech to text for JAWS
2. Go to the storybook for MultiComboBox
3. There are multiple repetitive announcements indicating the combo box list is multi-selection

### Expected Behaviour

Since "Contains Multi-Selectable Items list" announcement is made for the combo box, the "Multiple selection mode" announcement for all list items is repetitive and can be avoided.

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/fb9536fb-d86a-40fe-a08b-75a5ee6d8715)

![Image](https://github.com/user-attachments/assets/2b384ca9-eda2-4592-8911-194879a2d823)

### UI5 Web Components for React Version

2.9.0

### UI5 Web Components Version

2.9.0

### Browser

Edge

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
